google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/ConnectionOptionsTest.java

```diff
@@ -577,62 +576,6 @@ public void testSetCredentialsAndEncodedCredentials() throws Exception {
             "Cannot specify both a credentials URL and encoded credentials. Only set one of the properties.");
   }
 
-  @Test
```

Comment:
Is there an option to set automatic dialect detection to `false` or do you think we will not need that?

Reply:
`ConnectionOptions` does not have that option, and I also don't think we need that, as a connection must always know the dialect. So I think that if we were to add that option, then we probably also need to add a `setDialect` method (or the `setAutomaticDialectDetection(false)` must automatically also state something like 'this will make the connection always assume that the dialect is Google standard SQL).

As this is a class and not an interface, we can add those methods later if we find that it would be beneficial without introducing a breaking change.